gateway-ha/src/main/java/io/trino/gateway/ha/router/TrinoQueryProperties.java

```diff
@@ -356,107 +391,128 @@ private QualifiedName qualifyName(QualifiedName table)
     }
 
     @SuppressWarnings("unused")
+    @JsonProperty
```

Comment:
`@SuppressWarnings("unused")` is redundant now because it's called from tests. Same for others.